### PR TITLE
swaped to collapse component

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: components/HackneyDS/Collapse/Collapse.js [hunk 1]
```diff
@@ -9,9 +9,11 @@ export const Collapse = ({
   title,
   className = '',
   expanded,
+  triggerOnlyButton,
```
Comment (akeamwoodsND) [inline components/HackneyDS/Collapse/Collapse.js:12]: prop name doesnt make sense

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

File: components/HackneyDS/Collapse/Collapse.js (hunk 1 above)
Comment (mugnivenko) [reply components/HackneyDS/Collapse/Collapse.js:12]: how i should name it?

Comment (akeamwoodsND) [reply components/HackneyDS/Collapse/Collapse.js:12]: well what does it do?

**Comment by akeamwoodsND** [comment]
merge issues

File: components/HackneyDS/Collapse/Collapse.js (hunk 1 above)
Comment (mugnivenko) [reply components/HackneyDS/Collapse/Collapse.js:12]: image
For now, when you click on full container, it (expand, collapse) content. If you want, that (expand, collapse) work only for collapse button, you can use this boolean value
image

Comment (akeamwoodsND) [reply components/HackneyDS/Collapse/Collapse.js:12]: Okay, maybe something like buttonPointerEventsOnly

Comment (akeamwoodsND) [reply components/HackneyDS/Collapse/Collapse.js:12]: isButtonClickOnly

**Pull request merged**